### PR TITLE
Add mingw-w64 v10.0.0

### PR DESCRIPTION
**Pull request by mid-kid**

### CODE REVIEW

**Comment by mid-kid** [comment]
Fixes https://github.com/crosstool-ng/crosstool-ng/issues/1880

File: config/global/paths.in [hunk 1]
```diff
@@ -5,7 +5,7 @@ comment "Paths"
 config LOCAL_TARBALLS_DIR
     string
     prompt "Local tarballs directory"
-    default "${HOME}/src"
+    default "${HOME}/.local/opt/cross/src"
```
Comment (cpackham) [inline config/global/paths.in:8]: This shouldn't be included

File: maintainer/manage-packages.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ usage()
 $0 -- Test packages in crosstool-NG
 
 Verifies that the release tarballs can be downloaded for the packages
-available in crosstoo-NG and check that the patches apply cleanly.
+available in crosstool-NG and check that the patches apply cleanly.
```
Comment (cpackham) [inline maintainer/manage-packages.sh:11]: Ideally the typo fix would be a separate commit.

**Review by cpackham — CHANGES_REQUESTED** [review]

**Comment by cpackham** [comment]
Also the commit needs a reasonable message and sign-off.

**Review by cpackham — APPROVED** [review]

**Pull request merged**